README.md

```diff
@@ -274,55 +274,11 @@ Auth0
     }
 ```
 
-### Management API (Users)
+### Management API
 
-#### Retrieve user_metadata
+You can request more information than returned in the basic profile and manage the user's metadata. To do this, you can access the Auth0 [Management API](https://auth0.com/docs/api/management/v2)
```

Comment:
"than returned" ? sounds odd. Can you rephrase that?

Reply:
than is used when you're talking about comparisons. Also a cp from QS.